Library/Homebrew/shims/linux/super/cc

```diff
@@ -188,8 +188,8 @@ class Cmd
       args << "-L#{val}" if keep?(path) && @lset.add?(path)
     when /^-R(.+)?/
       # -R is not the correct way to pass rpaths to the linker
-      val  = chuzzle($1) || enum.next
-      wl = "-Wl," unless mode == :ld
+      path = chuzzle($1) || enum.next
+      wl   = "-Wl," unless mode == :ld
```

Comment:
I'm not a fan of this style of indentation. I know nearby code uses it, but I'd rather not propagate it.